packages/knip/src/plugins/eleventy/index.ts

```diff
@@ -13,14 +15,24 @@ const isEnabled: IsPluginEnabledCallback = ({ dependencies }) => hasDependency(d
 
 const ENTRY_FILE_PATTERNS = ['.eleventy.js', 'eleventy.config.{js,cjs}'];
 
-const PRODUCTION_ENTRY_FILE_PATTERNS = ['posts/**/*.11tydata.js', '_data/**/*.{js,cjs,mjs}'];
-
 const findEleventyDependencies: GenericPluginCallback = async (configFilePath, options) => {
   const { config } = options;
 
-  return config.entry
-    ? config.entry.map(toProductionEntryPattern)
-    : [...ENTRY_FILE_PATTERNS.map(toEntryPattern), ...PRODUCTION_ENTRY_FILE_PATTERNS.map(toProductionEntryPattern)];
+  let localConfig = await load(configFilePath);
+  if (typeof localConfig === 'function') localConfig = await localConfig(new DummyEleventyConfig());
+  localConfig = { ...localConfig, ...defaultEleventyConfig };
```

Comment:
Not sure whether this will work as intended, `defaultEleventyConfig` will always overwrite `localConfig` props?

Reply:
Oops yeah. Hmm wait I should merge this better anyway ahah, should I just do a lot of `localConfig.dir.input || defaultEleventyConfig.dir.input` or add something like `just-extend`?

Reply:
I'd prefer not to bring in a dependency for this. Maybe a mapper and/or a little helper function can do?